schemas/context/profile-lead-crm.schema.json

```diff
@@ -0,0 +1,131 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/profile-lead-crm",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "XDM Business Lead ",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/profile"
+  ],
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/context/profile-work-details",
+    "https://ns.adobe.com/xdm/context/profile-other-work-details",
+    "https://ns.adobe.com/xdm/context/profile-personal-details",
+    "https://ns.adobe.com/xdm/context/profile-person-details",
+    "https://ns.adobe.com/xdm/context/profile-owning-entities"
+  ],
+  "description": "This mixin is used to capture information regarding a B2B Lead.",
+  "definitions": {
+    "profile-lead-crm": {
+      "properties": {
+        "xdm:leadID": {
+          "title": "Lead Id",
+          "description": "Lead id.",
+          "type": "string"
+        },
+        "xdm:converted": {
+          "title": "Converted Indicator",
+          "description": "Indicates whether the Lead was successfully converted or not.",
+          "type": "boolean"
+        },
+        "xdm:convertedAccountID": {
+          "title": "Converted Account ID",
+          "description": "Account Id if the lead was successfully converted.",
+          "type": "string"
+        },
+        "xdm:convertedContactID": {
+          "title": "Converted Contact ID",
+          "description": "Contact Id if the lead was successfully converted.",
+          "type": "string"
+        },
+        "xdm:convertedDate": {
+          "title": "Converted Date",
+          "description": "Date of conversion if the lead was successfully converted.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "xdm:convertedOpportunityID": {
+          "title": "Converted Opportunity ID",
+          "description": "Opportunity Id if the lead was successfully converted.",
+          "type": "string"
+        },
+        "xdm:jigsawContactID": {
```

Comment:
It's already there as part of common. Remove it

Reply:
There are two fields. jigsaw and jigsawContactID. common has jigsaw.